templates/adp-backend-template-dotnet/skeleton/src/${{ values.dotnet_solution_name }}.Api/${{ values.dotnet_solution_name }}.Api.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<Version>0.1.1</Version>
+		<Nullable>enable</Nullable>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+		<DockerfileContext>.</DockerfileContext>
+		<DockerfileFile>..\..\Dockerfile</DockerfileFile>
+		<UserSecretsId>79971c91-7a80-4368-99e9-a67eb70d026b</UserSecretsId>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="8.1.0" />
+		<PackageReference Include="Azure.Identity" Version="1.11.3" />
+		<PackageReference Include="Azure.Monitor.OpenTelemetry.AspNetCore" Version="1.1.1" />
+		<PackageReference Include="Mapster" Version="7.4.0" />
```

Comment:
Do we definitely want to include mapster as a dependency? I feel like this template should be as un-opinionated as we can make it. Id never used mapster before joining this project before, all the defra projectes ive been on seemed to use automapper, but using a mapper in the first place brought up the occasional question about it even being needed

Reply:
See comment earlier .. I think some opinions are good - especially when dealing with lower skills devs.. balancing act

Reply:
As discussed in teams, I think this should be AutoMapper rather than mapster if we are going to include a mapping library, purely because its the most popular one, so its more likely people will have worked with it before.

Reply:
ok

Reply:
agreed